prow/cmd/hook/main.go

```diff
@@ -109,17 +110,25 @@ func main() {
 		logrus.WithError(err).Fatal("Error starting config agent.")
 	}
 
-	webhookSecretRaw, err := ioutil.ReadFile(o.webhookSecretFile)
+	secretAgent := &config.SecretAgent{}
+	if err := secretAgent.Start(o.webhookSecretFile, o.githubTokenFile); err != nil {
+		logrus.WithError(err).Fatal("Error starting secrets agent.")
+	}
+
+	sc := secretAgent.Config()
+
+	webhookSecret := bytes.TrimSpace(sc.SecretsMap[o.webhookSecretFile])
```

Comment:
Seems like we could trim space before storing so that end users of the agent don't need to do this